src/components/PageBanner/index.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const PageBanner = ({ imageUrl, imageTitle }) => (
-  <div
+const PageBanner = ({ imagesPath, imageFilename, imageExtension, imageTitle }) => (
+  <img
     className="coa-PageBanner"
-    style={{
-      backgroundImage: `url(${imageUrl})`,
-    }}
-    role="img"
+    srcset={`${imagesPath}/${imageFilename}.width-640${imageExtension} 640w,
+             ${imagesPath}/${imageFilename}.width-720${imageExtension} 720w,
+             ${imagesPath}/${imageFilename}.width-750${imageExtension} 750w,
+             ${imagesPath}/${imageFilename}.width-828${imageExtension} 828w,
+             ${imagesPath}/${imageFilename}.width-1080${imageExtension} 1080w,
+             ${imagesPath}/${imageFilename}.width-1440${imageExtension} 1440w,
+             ${imagesPath}/${imageFilename}.width-2160${imageExtension} 2160w,`}
```

Comment:
this list of sizes looks really similar to the list above in `HeroHome`. Is there any way we could abstract the sizes we want to use into an array that would make this better than copy/paste next time we need to use srcset this way in another component?